tests/local/network.go

```diff
@@ -174,11 +162,56 @@ func newLocalNetwork(warpGenesisFile string) *localNetwork {
 	}
 	res.setSubnetValues(subnetAID)
 	res.setSubnetValues(subnetBID)
-	res.setSubnetValues(subnetCID)
+	res.setPrimaryNetworkValues()
 	return res
 }
 
-func (n *localNetwork) setSubnetValues(subnetID ids.ID) {
+// Should be called after setSubnetValues for all subnets
+func (n *LocalNetwork) setPrimaryNetworkValues() {
+	// Get the C-Chain node URIs.
+	// All subnet nodes are also C-Chain nodes, so we can include them all here
```

Comment:
nit: can you elaborate on the comment to make it more clear that all subnet nodes validate the primary network, therefore are also c-chain nodes